js/phoenix/src/components/Button/Button.jsx

```diff
@@ -1,6 +1,6 @@
-export const Button = ({ text }) => {
+export const Button = ({ text, type }) => {
```

Comment:
@StephenJakku 
Remove type for button and send email password as props to onclick function on the button

Reply:
removed type, added new param handleClick to send the handleLogin function to Button component.